common/common-api/src/main/java/org/ow2/proactive/authentication/ConnectionInfo.java

```diff
@@ -102,4 +131,40 @@ public void setCredentialFile(File credentialFile) {
     public void setInsecure(boolean insecure) {
         this.insecure = insecure;
     }
+
+    /**
+     * Extract the Windows domain name from the full login
+     * parseDomain("domain\\user") returns domain
+     * parseDomain("user") returns null
+     * @param fullLogin the login to parse
+     * @return the domain name, null if no domain is specified
+     * @since Scheduling 3.0.1
+     */
+    public static String parseDomain(String fullLogin) {
+        if (fullLogin.contains("\\")) {
+            String domain = fullLogin.substring(0, fullLogin.indexOf("\\"));
+            if ("".equals(domain.trim())) {
+                return null;
+            }
```

Comment:
This block is useless. It is included in the instruction:
`return "".equals(domain.trim()) ? null : domain;`

Reply:
you're right, this code is a blind copy/paste